style.css

```diff
@@ -1,7 +1,11 @@
 body {
+    display: flex;
```

Comment:
I'd avoid changing the `display` of the whole body. I suggest doing that on the `wrapper` instead.